lib/workers/repository/init/merge.ts

```diff
@@ -27,12 +27,16 @@ export async function detectRepoFileConfig(): Promise<RepoFileConfig> {
   const cache = getCache();
   let { configFileName } = cache;
   if (configFileName) {
-    let configFileParsed = (await platform.getJsonFile(configFileName))!;
-    if (configFileParsed) {
-      if (configFileName === 'package.json') {
-        configFileParsed = configFileParsed.renovate;
+    try {
+      let configFileParsed = (await platform.getJsonFile(configFileName))!;
+      if (configFileParsed) {
+        if (configFileName === 'package.json') {
+          configFileParsed = configFileParsed.renovate;
+        }
+        return { configFileName, configFileParsed };
       }
-      return { configFileName, configFileParsed };
+    } catch (err) {
+      // probably file doesn't exist
```

Comment:
i think this can't really happen. the only scenario is:

- renovate fetches Head Sha
- user deletes renovate config
- renovate ties to fetch config file

in this case throw a repo changed error

Reply:
Happens during tests when repo not onboarded, we have same logic inside the isOnboarded function which has a try-catch block to prevent erros.